### ISSUE TITLE
[Windows] unable to resize/scale window

### ISSUE THREAD

**Issue by stutzjr** [Normalised/ProkDrumsEditor#5]
In Windows 10, I am unable to resize or rescale the window to fit my laptop display, thus the control bar on the bottom is never visible, and I always have to drag the window from partially offscreen on startup.  The native resolution on this older Acer laptop is 1366x768.

**Comment by stutzjr** [comment]
As a sub note of this issue, because I cannot see the control bar at the bottom, I am unable to check the functionality of the "IgnoreCV" control.  Does any state of the editor program track the state of the knobs on the front of the module?  The module seems to still respond to the knobs, but the editor screen does not update.

**Comment by Normalised** [comment]
I'll check and see how likely rescaling is, but lots of things are already very small, so rescaling them smaller would make them unusable.

To be honest I'm not sure how likely it is to be able to fit the current style of the editor into such a small space. I'll keep it in mind and see what ideas i can come up with.

Ignore CV : This stops the module from responding to CV control. this is for situations where you have the module cvs wired up and dont want to remove the cables and control the module from the editor.

Knob Tracking : No, the ui doesn't reflect knob position yet.

**Comment by stutzjr** [comment]
I don't think it is necessary to rescale the editor to fit in a smaller space.  All I'm asking is to consider changing the editor window's class from "modal" or "fixed" to "normal" so that a user could click the "maximise" button and the window can be automatically centered and made to fill the display, and add scroll bars if necessary.  I know that's not ideal but at least it is usable.

**Comment by Normalised** [comment]
Ok, i'll see what I can do.

**Comment by morbank** [comment]
Having the same issue on my older MacBook Pro (native res is 1280x800).  I'm unable to access the control bar at the bottom as well.
